src/fswAlgorithms/transDetermination/ephemDifferenceWithUncertainty/ephemDifferenceWithUncertainty.cpp

```diff
@@ -47,6 +47,46 @@ void EphemDifferenceWithUncertainty::Reset(uint64_t CurrentSimNanos)
  */
 void EphemDifferenceWithUncertainty::UpdateState(uint64_t CurrentSimNanos)
 {
+    EphemerisMsgPayload ephemBaseInBuffer = this->ephemBaseInMsg();
+    EphemerisMsgPayload ephemSecondaryInBuffer = this->ephemSecondaryInMsg();
+
+    // take timeTag from secondary, as timeTag from primary/base may be constant due to stand-alone message
+    double timeTag = ephemSecondaryInBuffer.timeTag;
+
+    /*! - compute relative states */
+    Eigen::Vector3d r_1_N = cArray2EigenVector3d(ephemBaseInBuffer.r_BdyZero_N);
+    Eigen::Vector3d v_1_N = cArray2EigenVector3d(ephemBaseInBuffer.v_BdyZero_N);
+    Eigen::Vector3d r_2_N = cArray2EigenVector3d(ephemSecondaryInBuffer.r_BdyZero_N);
+    Eigen::Vector3d v_2_N = cArray2EigenVector3d(ephemSecondaryInBuffer.v_BdyZero_N);
+
+    Eigen::Vector3d r_21_N = r_2_N - r_1_N;
+    Eigen::Vector3d v_21_N = v_2_N - v_1_N;
+
+    int num_states = 6;
```

Comment:
:large_orange_diamond: Change to camelCase.

Reply:
Done.